gammapy/scripts/spectrum_pipe.py

```diff
@@ -0,0 +1,86 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import logging
+from ..spectrum import (
+    SpectrumEnergyGroupMaker,
+    FluxPointEstimator,
+    SpectrumExtraction,
+    SpectrumFit,
+    SpectrumResult,
+)
+from ..background import ReflectedRegionsBackgroundEstimator
+
+__all__ = [
+    'SpectrumAnalysisIACT',
+]
+
+log = logging.getLogger(__name__)
+
+
+class SpectrumAnalysisIACT(object):
+    """High-level analysis class to perform a full 1D IACT spectral analysis
```

Comment:
Always end with a dot "." for the first summary line in docstrings. It's part of the Numpy docstring standard and the docstring linter (https://github.com/PyCQA/pydocstyle) points it out. I'll go through and fix our existing code soon, but let's get in the habit of doing it for new code like here.